### DISCUSSION TITLE
Banditry

### DISCUSSION THREAD

**Discussion by MasonFort** [TownyAdvanced/Towny#5947] [Ideas]
I believe a banditry system would be well-suited for Towny. The idea is that a townless player can become a bandit by killing a resident of a town through PvP. Bandits are unique, as they will automatically be labelled as outlaws in the originating city of any resident they kill and any city near the killing. However, bandits and only bandits are able to loot the "bodies" of the players they kill, stealing a portion of their balance. Bandit players are not allowed to join any town, regardless if they are an outlaw in that town or not. A bandit player can remove the 'bandit' tag after they are no longer an outlaw in any city and/or after a configurable amount of time.

**Comment by ItsJuls** [comment]
I assume this can be done through Towny API, since there is already an outlaw system and this could be done manually also

**Comment by LlmDl** [comment]
Aside from the part where only the bandit can loot the bodies, this could be made as a separate plugin using the API. Marking those drops when it is a banditry kill might be easy to do, making this something I could make for you. I don't think I'll be adding it directly to Towny.

**Reply by MasonFort** [reply]
When you say "making this something I could make for you", do you mean making the entire banditry plugin or just the part to mark the bodies? If you don't mean the entire plugin, do you have any recommendations within your community who would be willing for me to commission them to make it? Also, I just want to say thanks for continuing to develop Towny. I've been enjoying it ever since I started playing Minecraft as a kid.

**Reply by LlmDl** [reply]
I'd be willing to code the whole thing as an add-on jar probably, for a fee. I might have to do a bit of research about the drops-exclusive-pick-up to see if its possible first.

**Reply by LlmDl** [reply]
Looks like the exclusive-drops thing is also doable. If you're interested in getting this done contact me on the discord.

**Reply by MasonFort** [reply]
Sweet! I'll send over a message.

**Reply by MasonFort** [reply]
I can't send a message to you because we aren't friends and I can't add you so I think you'll have to add me. My discord is Mason Fort#0001
